pkgs/development/python-modules/rchitect/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cffi
+, six
+, pytest
+, pytest-mock
+, R
+, rPackages }:
+
+buildPythonPackage rec {
+  pname = "rchitect";
+  version = "0.3.40";
+
+  src = fetchFromGitHub {
+    owner = "randy3k";
+    repo = pname;
+    rev = "releases/tags/v${version}";
+    sha256 = "yJMiPmusZ62dd6+5VkA2uSjq57a0C3arG8CgiUUHKpk=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py --replace '"pytest-runner"' ""
+  '';
+
+  propagatedBuildInputs = [
+    cffi
+    six
+  ] ++ (with rPackages; [
+    reticulate
+  ]);
+
+  nativeCheckInputs = [
+    pytest
```

Comment:
```suggestion
    pytestCheckHook
```

Reply:
> Can we hardcode the path to `R` in https://github.com/randy3k/rchitect/blob/7c0d72aa3fbb5c7dc42ff3ae43c2346bd0ce5a39/rchitect/utils.py#L74?

We can do this, but does it matter? The package works perfectly even when `R` is not hardcoded and it makes no difference to the tests whether it is hardcoded or not.

Reply:
> The package works perfectly even when `R` is not hardcoded

Only if it's in your `$PATH`.

Reply:
Not necessarily. For example, within my own installation:

```{shell}
[genghiz@Work-PC ~/Workspace/nixos-config/packages/rchitect]% which R
R not found
[genghiz@Work-PC ~/Workspace/nixos-config/packages/rchitect]% radian
R version 4.2.1 (2022-06-23) -- "Funny-Looking Kid"
Platform: x86_64-pc-linux-gnu (64-bit)

r$> R.home()
[1] "/nix/store/madsjwak5p47x4yyhvxwnrx0d87rsm3k-R-4.2.1/lib/R"
```

The contents of the radian wrapper are:

```{bash}
#! /nix/store/zcla0ljiwpg5w8pvfagfjq1y2vasfix5-bash-5.1-p16/bin/bash -e
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/g1i9dz4mygrlj9bx27n6jqgkvw5a14sp-r-viridisLite-0.4.1/library'':'/':'}
R_LIBS_SITE='/nix/store/g1i9dz4mygrlj9bx27n6jqgkvw5a14sp-r-viridisLite-0.4.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/g631f3jwpmmjsp930ndvdmxm59g0jps8-r-RColorBrewer-1.1-3/library'':'/':'}
R_LIBS_SITE='/nix/store/g631f3jwpmmjsp930ndvdmxm59g0jps8-r-RColorBrewer-1.1-3/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/c1ijaswripc485vbx4qih7gv1gqiaynb-r-colorspace-2.0-3/library'':'/':'}
R_LIBS_SITE='/nix/store/c1ijaswripc485vbx4qih7gv1gqiaynb-r-colorspace-2.0-3/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/np942bzqx1qikdyrqrhblqy99hq8a14k-r-munsell-0.5.0/library'':'/':'}
R_LIBS_SITE='/nix/store/np942bzqx1qikdyrqrhblqy99hq8a14k-r-munsell-0.5.0/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/zdxlv83n4gh23vbzdap1g75bcxnbiiya-r-labeling-0.4.2/library'':'/':'}
R_LIBS_SITE='/nix/store/zdxlv83n4gh23vbzdap1g75bcxnbiiya-r-labeling-0.4.2/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/pp0kasq8v818lxgy7nbqccz7srr4vjav-r-farver-2.1.1/library'':'/':'}
R_LIBS_SITE='/nix/store/pp0kasq8v818lxgy7nbqccz7srr4vjav-r-farver-2.1.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/0xrc1rdnj0npghn05070sphczgf8nrr8-r-scales-1.2.1/library'':'/':'}
R_LIBS_SITE='/nix/store/0xrc1rdnj0npghn05070sphczgf8nrr8-r-scales-1.2.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/xa2y96i52gmviayqvazp2msacxq4jjzh-r-isoband-0.2.6/library'':'/':'}
R_LIBS_SITE='/nix/store/xa2y96i52gmviayqvazp2msacxq4jjzh-r-isoband-0.2.6/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/9l7xwjvkwhqwd3h52hspymw6rm80hx74-r-gtable-0.3.1/library'':'/':'}
R_LIBS_SITE='/nix/store/9l7xwjvkwhqwd3h52hspymw6rm80hx74-r-gtable-0.3.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/5gh4nx52vvn706g2llyval9kjbix9m9z-r-ggplot2-3.3.6/library'':'/':'}
R_LIBS_SITE='/nix/store/5gh4nx52vvn706g2llyval9kjbix9m9z-r-ggplot2-3.3.6/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/72a8pyim39bpnj24kjv0cfyslhr8f23d-r-tidyselect-1.2.0/library'':'/':'}
R_LIBS_SITE='/nix/store/72a8pyim39bpnj24kjv0cfyslhr8f23d-r-tidyselect-1.2.0/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/dj5zgpx7c5qmpsysw7h5r14c4z80nyvl-r-pkgconfig-2.0.3/library'':'/':'}
R_LIBS_SITE='/nix/store/dj5zgpx7c5qmpsysw7h5r14c4z80nyvl-r-pkgconfig-2.0.3/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/24w0vgk3q3h1fabz3j2mm992fv1y5jzn-r-tibble-3.1.8/library'':'/':'}
R_LIBS_SITE='/nix/store/24w0vgk3q3h1fabz3j2mm992fv1y5jzn-r-tibble-3.1.8/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/i6xa6c0l5vddzxpipzx78zvgncrvaawx-r-utf8-1.2.2/library'':'/':'}
R_LIBS_SITE='/nix/store/i6xa6c0l5vddzxpipzx78zvgncrvaawx-r-utf8-1.2.2/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/94qys8s30s5ayn28ylqddg4bq3rxmga9-r-fansi-1.0.3/library'':'/':'}
R_LIBS_SITE='/nix/store/94qys8s30s5ayn28ylqddg4bq3rxmga9-r-fansi-1.0.3/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/hkfliq6wxlh82wmxqy27va2gx3ici5h5-r-pillar-1.8.1/library'':'/':'}
R_LIBS_SITE='/nix/store/hkfliq6wxlh82wmxqy27va2gx3ici5h5-r-pillar-1.8.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/i0s62mx5yfbqyhvi7kld8q121qbv630k-r-generics-0.1.3/library'':'/':'}
R_LIBS_SITE='/nix/store/i0s62mx5yfbqyhvi7kld8q121qbv630k-r-generics-0.1.3/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/x8av3ix3yg0fzqzf86wygbpd4p0a24db-r-dplyr-1.0.10/library'':'/':'}
R_LIBS_SITE='/nix/store/x8av3ix3yg0fzqzf86wygbpd4p0a24db-r-dplyr-1.0.10/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/1awc2l3kbs4r33vqzhhcxc78braf82z2-r-lifecycle-1.0.3/library'':'/':'}
R_LIBS_SITE='/nix/store/1awc2l3kbs4r33vqzhhcxc78braf82z2-r-lifecycle-1.0.3/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/l68lg9c7dbkgbynp408fpvbjhraw51lk-r-vctrs-0.5.0/library'':'/':'}
R_LIBS_SITE='/nix/store/l68lg9c7dbkgbynp408fpvbjhraw51lk-r-vctrs-0.5.0/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/kblygfzw74mhind221md2z6w7p5nqsn9-r-R.utils-2.12.0/library'':'/':'}
R_LIBS_SITE='/nix/store/kblygfzw74mhind221md2z6w7p5nqsn9-r-R.utils-2.12.0/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/vk5zdrylwmrisn24crzi112q20ydydmz-r-R.oo-1.25.0/library'':'/':'}
R_LIBS_SITE='/nix/store/vk5zdrylwmrisn24crzi112q20ydydmz-r-R.oo-1.25.0/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/p3lzzfw8ndd8syv69qgf91bqw5lkvvhj-r-R.methodsS3-1.8.2/library'':'/':'}
R_LIBS_SITE='/nix/store/p3lzzfw8ndd8syv69qgf91bqw5lkvvhj-r-R.methodsS3-1.8.2/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/nimfkij7qwb7k24mybgkmn4rzza93jkh-r-R.cache-0.16.0/library'':'/':'}
R_LIBS_SITE='/nix/store/nimfkij7qwb7k24mybgkmn4rzza93jkh-r-R.cache-0.16.0/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/r5h3fmmfqp8pq4x8frhik2fypyk8d0yk-r-styler-1.8.0/library'':'/':'}
R_LIBS_SITE='/nix/store/r5h3fmmfqp8pq4x8frhik2fypyk8d0yk-r-styler-1.8.0/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/rgl5pdnvxw8g8qqi9c8aqki0bmbn1prr-r-purrr-0.3.5/library'':'/':'}
R_LIBS_SITE='/nix/store/rgl5pdnvxw8g8qqi9c8aqki0bmbn1prr-r-purrr-0.3.5/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/69jcxqlnp5d2cv4c68ybr2v28w7yl313-r-pkgload-1.3.1/library'':'/':'}
R_LIBS_SITE='/nix/store/69jcxqlnp5d2cv4c68ybr2v28w7yl313-r-pkgload-1.3.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/iwlby355wc1n5s4kz0f21v1qwh2ialsp-r-commonmark-1.8.1/library'':'/':'}
R_LIBS_SITE='/nix/store/iwlby355wc1n5s4kz0f21v1qwh2ialsp-r-commonmark-1.8.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/4f550rni2g7rhi2mg4shcyc91ahfp03k-r-brew-1.0-8/library'':'/':'}
R_LIBS_SITE='/nix/store/4f550rni2g7rhi2mg4shcyc91ahfp03k-r-brew-1.0-8/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/938b9xdhqvy321ijkaf2zq4hyz8dsqvb-r-roxygen2-7.2.1/library'':'/':'}
R_LIBS_SITE='/nix/store/938b9xdhqvy321ijkaf2zq4hyz8dsqvb-r-roxygen2-7.2.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/51lg48vm5lgzidvpk1nffgnpdqiddp73-r-xmlparsedata-1.0.5/library'':'/':'}
R_LIBS_SITE='/nix/store/51lg48vm5lgzidvpk1nffgnpdqiddp73-r-xmlparsedata-1.0.5/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/04z4qw29ynlxx2n65vqq2wl6f222qpm9-r-xml2-1.3.3/library'':'/':'}
R_LIBS_SITE='/nix/store/04z4qw29ynlxx2n65vqq2wl6f222qpm9-r-xml2-1.3.3/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/1m1f2v2w65f47dg7n02pc6clcy0p2430-r-lazyeval-0.2.2/library'':'/':'}
R_LIBS_SITE='/nix/store/1m1f2v2w65f47dg7n02pc6clcy0p2430-r-lazyeval-0.2.2/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/yyvr943w0v3n7b31inx0gwqy47nwdg7n-r-rex-1.2.1/library'':'/':'}
R_LIBS_SITE='/nix/store/yyvr943w0v3n7b31inx0gwqy47nwdg7n-r-rex-1.2.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/f50i7cyz293jpfn9vhn2glpk0sv0k7sr-r-yaml-2.3.6/library'':'/':'}
R_LIBS_SITE='/nix/store/f50i7cyz293jpfn9vhn2glpk0sv0k7sr-r-yaml-2.3.6/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/r0idy6avr3q0c8n2y2x8rwna19l2yjji-r-stringi-1.7.8/library'':'/':'}
R_LIBS_SITE='/nix/store/r0idy6avr3q0c8n2y2x8rwna19l2yjji-r-stringi-1.7.8/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/an8sacwnqhjhch9dkcwd6a5fwrgh3h59-r-magrittr-2.0.3/library'':'/':'}
R_LIBS_SITE='/nix/store/an8sacwnqhjhch9dkcwd6a5fwrgh3h59-r-magrittr-2.0.3/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/xqg50p80f8vhgha1k3dmdx4cky3kwxjg-r-stringr-1.4.1/library'':'/':'}
R_LIBS_SITE='/nix/store/xqg50p80f8vhgha1k3dmdx4cky3kwxjg-r-stringr-1.4.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/4ybf58wdi382hajd6pdybix4f4qh3fh8-r-xfun-0.34/library'':'/':'}
R_LIBS_SITE='/nix/store/4ybf58wdi382hajd6pdybix4f4qh3fh8-r-xfun-0.34/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/n63cq10zz1c1jjzn3rjk0vqrjcgyvzl3-r-highr-0.9/library'':'/':'}
R_LIBS_SITE='/nix/store/n63cq10zz1c1jjzn3rjk0vqrjcgyvzl3-r-highr-0.9/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/yzrx5n6nzpwamm0h2b0jdjhxs7p6rfc8-r-evaluate-0.17/library'':'/':'}
R_LIBS_SITE='/nix/store/yzrx5n6nzpwamm0h2b0jdjhxs7p6rfc8-r-evaluate-0.17/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/yglv1ajnia2342gf3d2fx4wl18hgp6sg-r-knitr-1.40/library'':'/':'}
R_LIBS_SITE='/nix/store/yglv1ajnia2342gf3d2fx4wl18hgp6sg-r-knitr-1.40/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/jd8sag3spnl15yhjc0qrp5ndlbd3c96q-r-glue-1.6.2/library'':'/':'}
R_LIBS_SITE='/nix/store/jd8sag3spnl15yhjc0qrp5ndlbd3c96q-r-glue-1.6.2/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/wq3840zxqczbry3m47pdg6cdpy4j3brx-r-digest-0.6.30/library'':'/':'}
R_LIBS_SITE='/nix/store/wq3840zxqczbry3m47pdg6cdpy4j3brx-r-digest-0.6.30/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/jnzmnxk5lbpxdf4wyg7c0c4xxyhkics3-r-remotes-2.4.2/library'':'/':'}
R_LIBS_SITE='/nix/store/jnzmnxk5lbpxdf4wyg7c0c4xxyhkics3-r-remotes-2.4.2/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/k1r8aianhibvh4d9gww30jy5hjmrqkdv-r-cli-3.4.1/library'':'/':'}
R_LIBS_SITE='/nix/store/k1r8aianhibvh4d9gww30jy5hjmrqkdv-r-cli-3.4.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/5qnpxvrn8aiq5p296d5lyyrfcd8prlz2-r-desc-1.4.2/library'':'/':'}
R_LIBS_SITE='/nix/store/5qnpxvrn8aiq5p296d5lyyrfcd8prlz2-r-desc-1.4.2/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/i7m066mfnhy2i5giz5d5pkzd8xdvsv5g-r-cyclocomp-1.1.0/library'':'/':'}
R_LIBS_SITE='/nix/store/i7m066mfnhy2i5giz5d5pkzd8xdvsv5g-r-cyclocomp-1.1.0/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/xx5lqpgw9qzccscjrlxyasishf55lla2-r-crayon-1.5.2/library'':'/':'}
R_LIBS_SITE='/nix/store/xx5lqpgw9qzccscjrlxyasishf55lla2-r-crayon-1.5.2/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/jps0d36ixkcyphinv1n864a06ipb4d4v-r-backports-1.4.1/library'':'/':'}
R_LIBS_SITE='/nix/store/jps0d36ixkcyphinv1n864a06ipb4d4v-r-backports-1.4.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/3h9jcv2gz5lhsj74jbib1nyn8aijs264-r-lintr-3.0.2/library'':'/':'}
R_LIBS_SITE='/nix/store/3h9jcv2gz5lhsj74jbib1nyn8aijs264-r-lintr-3.0.2/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/5djhhkyrgrc75pvlh2i3zly0d73r7cf8-r-fs-1.5.2/library'':'/':'}
R_LIBS_SITE='/nix/store/5djhhkyrgrc75pvlh2i3zly0d73r7cf8-r-fs-1.5.2/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/ac9arkhxail55172j25qv0mvsphx0v07-r-collections-0.3.6/library'':'/':'}
R_LIBS_SITE='/nix/store/ac9arkhxail55172j25qv0mvsphx0v07-r-collections-0.3.6/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/34h9csxwa32i92756987gk853vc5w8pn-r-R6-2.5.1/library'':'/':'}
R_LIBS_SITE='/nix/store/34h9csxwa32i92756987gk853vc5w8pn-r-R6-2.5.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/s83rypb49mr1g7iwhns00lb2vnd33qs7-r-ps-1.7.2/library'':'/':'}
R_LIBS_SITE='/nix/store/s83rypb49mr1g7iwhns00lb2vnd33qs7-r-ps-1.7.2/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/gzvrcsjl8yh227k4gd8zjvkyf8vwz7my-r-processx-3.8.0/library'':'/':'}
R_LIBS_SITE='/nix/store/gzvrcsjl8yh227k4gd8zjvkyf8vwz7my-r-processx-3.8.0/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/953m9xw5nmm0w8ihw8swsgzrhnlgafv9-r-callr-3.7.2/library'':'/':'}
R_LIBS_SITE='/nix/store/953m9xw5nmm0w8ihw8swsgzrhnlgafv9-r-callr-3.7.2/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/29ynlkayw4lnkiz5268yjky1ii7z07w4-r-languageserver-0.3.14/library'':'/':'}
R_LIBS_SITE='/nix/store/29ynlkayw4lnkiz5268yjky1ii7z07w4-r-languageserver-0.3.14/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/crk85gyx54p436s7g33f430inr0c0l1p-r-systemfonts-1.0.4/library'':'/':'}
R_LIBS_SITE='/nix/store/crk85gyx54p436s7g33f430inr0c0l1p-r-systemfonts-1.0.4/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/q5pw1qfajdzabsr2b5d5438aiiskk8qi-r-rlang-1.0.6/library'':'/':'}
R_LIBS_SITE='/nix/store/q5pw1qfajdzabsr2b5d5438aiiskk8qi-r-rlang-1.0.6/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/z4mjhqs5x4613gv0rx1l65734jlqz6kg-r-later-1.3.0/library'':'/':'}
R_LIBS_SITE='/nix/store/z4mjhqs5x4613gv0rx1l65734jlqz6kg-r-later-1.3.0/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/87lxl858vvf2rf54p0ycb417hylj7zzd-r-cpp11-0.4.3/library'':'/':'}
R_LIBS_SITE='/nix/store/87lxl858vvf2rf54p0ycb417hylj7zzd-r-cpp11-0.4.3/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/6igvimx6kxgz41vzw3ncqsanwfyv7bkm-r-BH-1.78.0-0/library'':'/':'}
R_LIBS_SITE='/nix/store/6igvimx6kxgz41vzw3ncqsanwfyv7bkm-r-BH-1.78.0-0/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/5qcyqn79apzg6myrwhh3pxav10jz5hpr-r-httpgd-1.3.0/library'':'/':'}
R_LIBS_SITE='/nix/store/5qcyqn79apzg6myrwhh3pxav10jz5hpr-r-httpgd-1.3.0/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/x7b5gj0azca38p9bfwrhhwk66249fna8-r-renv-0.16.0/library'':'/':'}
R_LIBS_SITE='/nix/store/x7b5gj0azca38p9bfwrhhwk66249fna8-r-renv-0.16.0/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/8dwv590p2l21q85dg4495i695d4pzxjs-r-survival-3.4-0/library'':'/':'}
R_LIBS_SITE='/nix/store/8dwv590p2l21q85dg4495i695d4pzxjs-r-survival-3.4-0/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/9z5nr8l481pn067qzfi7s2s0hhzpjhsi-r-spatial-7.3-15/library'':'/':'}
R_LIBS_SITE='/nix/store/9z5nr8l481pn067qzfi7s2s0hhzpjhsi-r-spatial-7.3-15/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/w9pj7qpfnvkgjyigwf78fvkk3wpbdzdg-r-rpart-4.1.19/library'':'/':'}
R_LIBS_SITE='/nix/store/w9pj7qpfnvkgjyigwf78fvkk3wpbdzdg-r-rpart-4.1.19/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/05g160zswfhnlq0as488f11dmgypjyk1-r-nnet-7.3-18/library'':'/':'}
R_LIBS_SITE='/nix/store/05g160zswfhnlq0as488f11dmgypjyk1-r-nnet-7.3-18/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/lk35lg06a5nk8ps0bzz37aliszfhpcf7-r-nlme-3.1-160/library'':'/':'}
R_LIBS_SITE='/nix/store/lk35lg06a5nk8ps0bzz37aliszfhpcf7-r-nlme-3.1-160/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/18g1nk927bb1phx1f2a1vh4f7hkd27b2-r-mgcv-1.8-41/library'':'/':'}
R_LIBS_SITE='/nix/store/18g1nk927bb1phx1f2a1vh4f7hkd27b2-r-mgcv-1.8-41/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/m4akhgh1z8ggrgd201x797bvzbqfbkcj-r-KernSmooth-2.23-20/library'':'/':'}
R_LIBS_SITE='/nix/store/m4akhgh1z8ggrgd201x797bvzbqfbkcj-r-KernSmooth-2.23-20/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/x16xf5i366r8d8hyfyp2zqsmqq7lapf9-r-foreign-0.8-83/library'':'/':'}
R_LIBS_SITE='/nix/store/x16xf5i366r8d8hyfyp2zqsmqq7lapf9-r-foreign-0.8-83/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/v1gqppvz6vmzj2nb26sz8q190hhjz6vr-r-codetools-0.2-18/library'':'/':'}
R_LIBS_SITE='/nix/store/v1gqppvz6vmzj2nb26sz8q190hhjz6vr-r-codetools-0.2-18/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/99dh2j9jmz3c45ggwv8r4ak2zsnj6z00-r-cluster-2.1.4/library'':'/':'}
R_LIBS_SITE='/nix/store/99dh2j9jmz3c45ggwv8r4ak2zsnj6z00-r-cluster-2.1.4/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/4kj1kzwjy2hjwvzj48mc65pshpjljc06-r-MASS-7.3-58.1/library'':'/':'}
R_LIBS_SITE='/nix/store/4kj1kzwjy2hjwvzj48mc65pshpjljc06-r-MASS-7.3-58.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/w39j7ab5g526qcxc2jrgbcl2asc37kp0-r-class-7.3-20/library'':'/':'}
R_LIBS_SITE='/nix/store/w39j7ab5g526qcxc2jrgbcl2asc37kp0-r-class-7.3-20/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/kiz9xj8cb7xdig0xs1qwjqibc9s0w2m1-r-boot-1.3-28/library'':'/':'}
R_LIBS_SITE='/nix/store/kiz9xj8cb7xdig0xs1qwjqibc9s0w2m1-r-boot-1.3-28/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/lbiq68zdd680hr79hp33171iw8m0zd4k-r-sys-3.4.1/library'':'/':'}
R_LIBS_SITE='/nix/store/lbiq68zdd680hr79hp33171iw8m0zd4k-r-sys-3.4.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/80vj8rlg58ys5bxs76ra8cyx644jn8l9-r-askpass-1.1/library'':'/':'}
R_LIBS_SITE='/nix/store/80vj8rlg58ys5bxs76ra8cyx644jn8l9-r-askpass-1.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/b23pnwhanvi2q6a6mmim6dbh4niws8pb-r-withr-2.5.0/library'':'/':'}
R_LIBS_SITE='/nix/store/b23pnwhanvi2q6a6mmim6dbh4niws8pb-r-withr-2.5.0/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/a89bb4wk9pc5gqkbmc34jizsz0zqgbc8-r-RcppTOML-0.1.7/library'':'/':'}
R_LIBS_SITE='/nix/store/a89bb4wk9pc5gqkbmc34jizsz0zqgbc8-r-RcppTOML-0.1.7/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/4a4and1955r75madp9shg0a8y05silz1-r-Rcpp-1.0.9/library'':'/':'}
R_LIBS_SITE='/nix/store/4a4and1955r75madp9shg0a8y05silz1-r-Rcpp-1.0.9/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/mwsib8pv54v166bd9mnnsw9ik613rc9x-r-rappdirs-0.3.3/library'':'/':'}
R_LIBS_SITE='/nix/store/mwsib8pv54v166bd9mnnsw9ik613rc9x-r-rappdirs-0.3.3/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/10vkh5h1a0yyxfk897dq7ckjcj7kh4gv-r-png-0.1-7/library'':'/':'}
R_LIBS_SITE='/nix/store/10vkh5h1a0yyxfk897dq7ckjcj7kh4gv-r-png-0.1-7/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/9w3j1ni6ifslzj2gdkaffix9rs9b1932-r-lattice-0.20-45/library'':'/':'}
R_LIBS_SITE='/nix/store/9w3j1ni6ifslzj2gdkaffix9rs9b1932-r-lattice-0.20-45/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/a3h1nv4ipfyvxffpcfcyij7fq6zasalj-r-Matrix-1.5-1/library'':'/':'}
R_LIBS_SITE='/nix/store/a3h1nv4ipfyvxffpcfcyij7fq6zasalj-r-Matrix-1.5-1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/7ic178n01z9nf6iwx7x0nfw4pj0y50gf-r-jsonlite-1.8.3/library'':'/':'}
R_LIBS_SITE='/nix/store/7ic178n01z9nf6iwx7x0nfw4pj0y50gf-r-jsonlite-1.8.3/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/c600j0bnllv8g0whvblj0apcmq9vkqr7-r-rprojroot-2.0.3/library'':'/':'}
R_LIBS_SITE='/nix/store/c600j0bnllv8g0whvblj0apcmq9vkqr7-r-rprojroot-2.0.3/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/9h9jgbqvzzfsh39g8sgrkbqwc9fcfvjr-r-here-1.0.1/library'':'/':'}
R_LIBS_SITE='/nix/store/9h9jgbqvzzfsh39g8sgrkbqwc9fcfvjr-r-here-1.0.1/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE:+':'$R_LIBS_SITE':'}
R_LIBS_SITE=${R_LIBS_SITE/':''/nix/store/fbabqfgw3kjlvd0lv9ll1161ypnq7rkc-r-reticulate-1.26/library'':'/':'}
R_LIBS_SITE='/nix/store/fbabqfgw3kjlvd0lv9ll1161ypnq7rkc-r-reticulate-1.26/library'$R_LIBS_SITE
R_LIBS_SITE=${R_LIBS_SITE#':'}
R_LIBS_SITE=${R_LIBS_SITE%':'}
export R_LIBS_SITE
exec "/nix/store/bfm6qcq1afwi3p9xfb4ass4zfyk8lnkr-python3.10-radian-0.6.4/bin/radian"  "$@"
```

Reply:
I see, because of the R libraries in `propagatedBuildInputs`. Do they need to be there? Can we hardcode their paths using a patch? Maybe using `rWrapper`?

Reply:
The libraries need to be there because they're called during the tests. This is how `rpy2` also handles this situation. They can be put in `buildInputs` as well (as is done by `rpy2`), but putting them in `nativeCheckInputs` doesn't work.

I've tried using `rWrapper` and it doesn't work either.

Reply:
There's also `checkInputs`.

Reply:
It doesn't work either. It's a rather vexing problem. I've tried this with multiple approaches and none of them seem to work except for this one.

Reply:
It's quite interesting. If you run `checkPhase` with `python3 -m pytest` it fails to import anything from `rchitect._cffi`. This is the error it generates:

```
> running install tests
> ImportError while loading conftest '/build/source/tests/conftest.py'.
> tests/conftest.py:2: in <module>
>     from rchitect import rcall
> rchitect/__init__.py:1: in <module>
>     from .setup import init, loop
> rchitect/setup.py:6: in <module>
>     from rchitect._cffi import ffi, lib
> E   ModuleNotFoundError: No module named 'rchitect._cffi'
```

If you run `pytest` directly this is not an issue. And since `pytestCheckHook` is the former, it just fails to run through the hook.

Reply:
@dotlambda Any other ideas? I've tried a few other things but I keep coming back to this solution.

Reply:
Try removing rchitect from pwd or prepending the sitePackages in $out to PYTHONPATH. The issue here looks like it prefers the source directory which does not have the compiled objects at the right location.

Why use the wrapper? It makes it way easier to disable broken by design tests.

Reply:
This has also been done. Thanks for the tip.